versioned/storage/dynamodb/src/main/java/org/projectnessie/versioned/storage/dynamodb/DynamoDBBackendTestFactory.java

```diff
@@ -40,9 +40,21 @@ public String getName() {
     return DynamoDBBackendFactory.NAME;
   }
 
+  @SuppressWarnings("ClassEscapesDefinedScope")
```

Comment:
IntelliJ flags this annotation as redundant... Could you double check, please?

Reply:
I've removed it. Although interestingly my IntilliJ complains when I remove the annotation.

Reply:
What does it say? My EAP 2 build is happy :)

Reply:
I guess it might be complaining about returning the package-private `DynamoDBBackend` type, but the interface defines the return value as `Backend`... I think the code is correct and IJ was overzealous :)

Reply:
Correction: I was running my old IJ built for this review before and it did not give that warning. I switched to EAP 3 (just now) and it does indeed issue the warning. Having considered the code carefully, I think the warning is justified.

I also see some usages that rely on the more specific return type... so cleaning this up is probably beyond the scope of this PR. Let's not suppress the warning though as a reminder to revisit this.

Reply:
We have the same issue in BigTable impl... we return `Backend` there, not `BigTableBackend`.